r2/r2/controllers/oembed.py

```diff
@@ -77,6 +77,12 @@ def _oembed_post(thing, **embed_options):
         time = datetime.now(g.tz).isoformat()
         live = 'data-card-created="{}"'.format(time)
 
+    script = ''
+    if embed_options.get('omitscript') == "false":
```

Comment:
You won't want a string comparison here, you'll want a boolean comparison (because `VBoolean` converts down below), and I think the logic here needs to support the default case.

You probably want `if not embed_options.get('omitscript', False):`, which will also handle cases where it's not passed in and should default to `False`.